spec/lib/vcr/errors_spec.rb

```diff
@@ -72,12 +72,20 @@ def request_with(values)
           end
         end
 
-        it 'mentions the details about the current casette' do
+        it 'mentions the details about the single cassette when there is one cassette' do
```

Comment:
Line is too long. [89/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.